Podfile.lock

```diff
@@ -73,4 +73,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 6ee5bcd18f0d60a4b82be64479784270153e3f26
 
-COCOAPODS: 1.1.0
+COCOAPODS: 1.0.1
```

Comment:
How do you want to resolve this? Should you update to 1.1.0 or should I see if I can downgrade? Maybe there's a way to create a virtualenv?